<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>geometry</class>
 <widget class="QWidget" name="geometry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(155, 192, 226);
border:0px;
border-top-right-radius: 25px;
border-bottom-right-radius: 25px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="exams_Icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/images/exams_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="program_Icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/images/program_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="results_Icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/images/results_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="inbox_Icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/images/inbox_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="profile_Icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/images/profile.icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logout_Icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>630</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/assets/images/log_out_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="logoutButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>620</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Allerta&quot;;
color:white;
text-align:left;</string>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="examsButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>195</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Allerta&quot;;
color:white;
text-align:left;</string>
    </property>
    <property name="text">
     <string>Exams</string>
    </property>
   </widget>
   <widget class="QPushButton" name="programButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;sadas&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Allerta&quot;;
color:white;
text-align:left;</string>
    </property>
    <property name="text">
     <string>Program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resultsButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Allerta&quot;;
color:white;
text-align:left;</string>
    </property>
    <property name="text">
     <string>Results</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inboxButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Allerta&quot;;
color:white;
text-align:left;</string>
    </property>
    <property name="text">
     <string>Inbox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="profileButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>86</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Allerta&quot;;
color:white;
text-align:left;</string>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QLabel" name="navigationLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 17pt;
color:white;</string>
    </property>
    <property name="text">
     <string>NAVIGATION</string>
    </property>
   </widget>
   <widget class="QLabel" name="logoIcon">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>-10</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/assets/images/Logo1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 600 12pt &quot;Crimson Text Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;UNIVERSE&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="backgroundblue_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #4486c3;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>671</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font-weight:bold;
font: 55pt &quot;Aura&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;GEOMETRY&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="exam1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>801</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(155, 192, 226);
border:7px solid #3F7DB5;
border-radius:35px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="exam1_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>801</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(155, 192, 226);
border:7px solid #3F7DB5;
border-radius:35px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="exam1_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>801</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(155, 192, 226);
border:7px solid #3F7DB5;
border-radius:35px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>backgroundblue_bg</zorder>
  <zorder>frame_2</zorder>
  <zorder>titleLabel</zorder>
  <zorder>exam1</zorder>
  <zorder>exam1_2</zorder>
  <zorder>exam1_3</zorder>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
